granian/cli.py

```diff
@@ -242,6 +274,10 @@ def cli(
     respawn_failed_workers: bool,
     respawn_interval: float,
     reload: bool,
+    reload_paths: List[pathlib.Path],
```

Comment:
Maybe this should be an `Optional[...]` as well, we can customise the display of the default value in the CLI.